index.html

```diff
@@ -0,0 +1,52 @@
+<link rel="stylesheet" href="css/style.css">
+<link href="https://fonts.googleapis.com/css?family=Poppins:200i,300,400&amp;display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+  <nav id="navbar">
+    
+       <a href="#welcome-section">About</a>
+       <a href="#projects">Work</a>
+       <a href="#conact">Contact</a>
+    
+
+  </nav>
+  
+<script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+<div id="welcome-section">
+  <h1>Hey I am Mimic</h1>
+  <h4 id="head4">a web developer</h4>
+</div>
+
+<div id="projects">
+  <div>
+    <h2 class="project-title">These are some of my projects</h2>
+  </div>
+  <div id="section">
+    <div id="random">
+      <a href="https://codepen.io/freeCodeCamp/full/qRZeGZ" target="_blank">
+        <div>
+          <img class="image" src="images/random-quote-machine.png" alt="img">
+          <p class="project-para">
+            <span class="code">&lt;</span>
+            Random Quote Machine
+            <span class="code">/&gt;</span>
+          </p>
+        </div></a>
+    </div>
+  <div id="first-link">
+    <a href="https://codepen.io/freeCodeCamp/full/zNqgVx"> 
+      <img class="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg" alt="project1">
+      <p class="project-name">
+        <span class="code">&lt;</span>
+        Tribute Page 
+        <span class="code">&#47;&gt;</span>
+      </p>
+    </a>
+  </div>
+</div>
```

Comment:
good job noticing this error 👍 however I think it would be better to phrase the comment more politely in order to keep the communication with the student more friendly, please refer to my previous comment for an example.